File: Back/src/main/kotlin/com/ssafy/tp/model/Review.kt
package com.ssafy.tp.model

import javax.persistence.*

@Entity(name = "review")
data class Review(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id")
    var id: Long,

    @Column(name = "content")
    var content:String,

    @Column(name = "product_id")
    var productId:Long,

    @Column(name = "member_id")
    var memberId:Long,
) {


}

File: Flutter/lib/model/Review.dart
class Review {
  int? id;
  String? content;
  int? productId;
  String? memberId;

  Review({
    this.id,
    this.content,
    this.productId,
    this.memberId,
  });

  factory Review.fromJson(Map<String, dynamic> json) {
    return Review(
        id: json['id'],
        content: json['content'],
        productId: json['productId'],
        memberId: json['memberId']);
  }

  Map<String, dynamic> toJson() => {
        'id': id,
        'content': content,
        'productId': productId,
        'memberId': memberId,
      };
}
